protocols/migration/migration_reports_6_0_0_to_reports_5_0_0.py

```diff
@@ -197,7 +197,7 @@ def migrate_interpretation_request_cancer(self, old_instance):
         if new_instance.cancerParticipant and new_instance.cancerParticipant.tumourSamples:
             samples = new_instance.cancerParticipant.tumourSamples
             for sample in samples:
-                if sample.diseaseType in (diseaseType.ENDOCRINE, diseaseType.OTHER):
+                if sample.diseaseType == diseaseType.ENDOCRINE:
```

Comment:
@priesgo I'm not sure this is right

Reply:
fixed in https://github.com/genomicsengland/GelReportModels/pull/330/commits/6522eda5d67885c18c4f230a8d84fe5581336d76